main.py

```diff
@@ -6,5 +6,7 @@
     print(letters_phone_number)
     phone_object = Phone(phone_letters=letters_phone_number)
     phone_is_invalid = not phone_object.validate_phone()
+    # if phone_is_invalid:
+    #     print("Your phone number is invalid.")
```

Comment:
Oops, it seems that I had made a small mistake here, could you uncomment these lines, please.